sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -973,13 +973,13 @@ public void beginRecognizeInvoicesWithOptions() throws IOException {
     }
 
     /**
-     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeIdDocumentsFromUrl(String)}
+     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeIdentityDocumentsFromUrl(String)}
      */
     public void beginRecognizeIDDocumentFromUrl() {
```

Comment:
idk if it is important for Java to fix the name in the tests too. just pointing out

Reply:
It is not so important on the codesnippet method name for but I will update it for consistency.